test/device/intrinsics.jl

```diff
@@ -157,6 +157,11 @@ end
     bufferE = MtlArray(e)
     vecE = Array(SpecialFunctions.erfc.(bufferE))
     @test vecE ≈ SpecialFunctions.erfc.(e)
+
+    f = collect(LinRange(nextfloat(-0.3f0), 0.3f0, 20))
```

Comment:
```suggestion
    f = collect(LinRange(-1f0, 1f0, 20))
```
Might as well test the whole domain including the extremes.